jumpscale/packages/marketplace/chats/peertube.py

```diff
@@ -293,13 +112,35 @@ def reservation(self):
         if not success:
             raise StopChatFlow(f"Failed to deploy workload {self.resv_id}")
 
-    @chatflow_step(title="Success", disable_previous=True)
-    def peertube_access(self):
-        res = f"""\
+        _id = deployer.expose_and_create_certificate(
+            pool_id=self.pool_info.reservation_id,
+            gateway_id=self.gateway.node_id,
+            network_name=self.network_view.name,
+            trc_secret=self.secret,
+            domain=self.domain,
+            email=self.user_info()["email"],
+            solution_ip=self.ip_address,
+            solution_port=80,
+            enforce_https=True,
+            test_cert=True,
```

Comment:
I think we can make the default `test_cert=False`
= True only for debugging purposes